desktop-src/CoreAudio/loopback-recording.md

```diff
@@ -68,7 +68,9 @@ WASAPI supports loopback recording regardless of whether the audio hardware cont
 
 Windows Vista provides digital rights management (DRM). Content providers rely on DRM to protect their proprietary music or other content from unauthorized copying and other illegal uses. Similarly, a trusted audio driver does not permit a loopback device to capture digital streams that contain protected content. Windows Vista allows only trusted drivers to play protected content. For more information about trusted drivers and DRM, see the Windows DDK documentation.
 
-WASAPI loopback contains the mix of all audio being played, regardless of the Terminal Services session the audio originated from. For example, you can run a loopback client in a service running in session 0 and capture audio from all user sessions, as well as audio being played from session 0.
+WASAPI loopback by default contains the mix of all audio being played, regardless of the Terminal Services session the audio originated from. For example, you can run a loopback client in a service running in session 0 and capture audio from all user sessions, as well as audio being played from session 0.
+
+Starting in Windows 10 Build 20348 you can include or exclude specific processes and their children from loopback capture by passing an [AUDIOCLIENT_ACTIVATION_PARAMS](https://learn.microsoft.com/windows/win32/api/audioclientactivationparams/ns-audioclientactivationparams-audioclient_activation_params) in the call to IMMDevice::Activate. See the [Application loopback audio capture sample](https://learn.microsoft.com/samples/microsoft/windows-classic-samples/applicationloopbackaudio-sample/).
```

Comment:
@zhuman Can you change these to site-relative links? Just remove "https://learn.microsoft.com" and start the link target with "/windows/...". Our build validation tools will flag these absolute links. Thanks!